r2/r2/lib/jsontemplates.py

```diff
@@ -501,6 +501,14 @@ def thing_attr(self, thing, attr):
         else:
             return SubredditJsonTemplate.thing_attr(self, thing, attr)
 
+class SubredditSidebarJsonTemplate(SubredditJsonTemplate):
+    _data_attrs_ = dict(
+        usertext_md="description",
+        usertext_html="description_html",
```

Comment:
What was your reasoning behind naming these `usertext_md` and `usertext_html`?  `description` and `description_html` provide consistency with the field names in /r/subreddit/about.json.